### ISSUE TITLE
Error alert component ("aria-hidden: true")

### ISSUE THREAD

**Issue by angelina-smith** [GSA/px-benefit-finder#1115]
### What's expected

Location: Make an error by not answering required form fields and review Error alerts.

1. We expect the "Error" alerts to be read by screen reader and for "aria-hidden: true" to be removed.

### What happened

1. "aria-hidden: true" is on the alert component and should be removed so that assistive technology is aware the component is there.

### How to replicate

Bugs are seen on Storybook and on live site.

**Comment by angelina-smith** [comment]
@fongcindy

**Comment by scottqueen-bixal** [comment]
@angelina-smith this may just need to be reviewed again for confirmation that the issue was fixed in the previous sprint.

the state of `aria-hidden` on the top level form alert should be as follows

> default: `aria-hidden="true"
 
This prevents the alert from appearing to assistive tech

> error: `aria-hidden="false"`

this exposes the alert to assistive technology

> error resolved: `aria-hidden="false"`

if all errors are resolved the attribute should revert back to hidden an not be displayed in the UI preventing it from being accessed by assistive technology


As it stands this is the current behavior observed on production.  Can you confirm if this is working as expected?

**Comment by scottqueen-bixal** [comment]
I've found two other locations where alerts are used and may need to be in the tabIndex,

Inside the accordions,

![Image](https://github.com/GSA/px-benefit-finder/assets/37077057/4eee579d-536e-4b04-a1aa-02af73247d30)

Here I am already seeing that axe is throwing an error, it expects it to be focusable

And Here in the intro, these should be focusable as well? Or, should they be different roles entirely?

![Image](https://github.com/GSA/px-benefit-finder/assets/37077057/472472fa-a57c-4b1f-aaa6-1e7cfe0976e6)

**Comment by angelina-smith** [comment]
<H1>Error alert</h1>
@scottqueen-bixal I'm able to hear the Error alerts being read by the screen reader! However, I noticed::

Location: "Questions 1 of 2" page
How to replicate: Leave all fields empty and press Continue.
What is occurring: The order of the error alerts are read out of order than what is visually presented.
"alert   Date should follow the following format January 19 2000. Error heading level 4, Complete all required fields" (See below NVDA screenviewer log)


![Image](https://github.com/GSA/px-benefit-finder/assets/142342010/a2fb3fea-4248-40f3-941f-2f3bc1b814cc)

Just to note, when move forward with #1121 , this will become irrelevant (the order of which multiple errors are read since they will be combined in a summary).

<h1> Informative status alerts</h1>
As for the other two places where alerts are being used, was there a reason we implemented the 'Informative status' alerts in the accordions and in the intro @fongcindy ? Curious because I'm looking at the USDWDS guidance for the alert component and I'm not sure the alerts in these 2 places fit the usage guidance


![Image](https://github.com/GSA/px-benefit-finder/assets/142342010/d9a97e9f-c11d-41d9-b126-6b81be89cb03)

- Feedback from AT UT sessions suggested the disclaimer list on the intro page just be coded as a bulleted list.
- In the accordion, I recommend us removing the alert component and having the text be normal content underneath the bulleted list.

Thoughts? @scottqueen-bixal @amycole501 @fongcindy

**Comment by scottqueen-bixal** [comment]
following bug issues ceated to address issues found with other alerts in this ticket
[Info alerts in accordions are hidden to AT](https://github.com/GSA/px-benefit-finder/issues/1183)	https://github.com/GSA/px-benefit-finder/issues/1183
[Notices list should not include informational alerts](https://github.com/GSA/px-benefit-finder/issues/1182)	https://github.com/GSA/px-benefit-finder/issues/1182
